HelloWorld/HelloWorld/Program.cs

```diff
@@ -10,8 +10,8 @@ internal class Program
 	{
 		private static void Main(string[] args)
 		{
-			Console.WriteLine("Hello World!");
+			Console.WriteLine("G'day mate!");
```

Comment:
Are we sure this is correct? I've been to Australia, and I've never heard anyone there say this.